### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by tanishq-arya**
## Related Issue or bug
  - Modified About content
  - Increased heading size
  - Added steps to fork and clone in installation

Fixes: #338 

#### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot**  | <b>updated screenshot </b> |
 
 
 
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,34 +27,65 @@ Backend
...
+8. Run the server
+```bash
 python manage.py runserver
 ```
-## First time to [![Open Source Love](https://badges.frapsoft.com/os/v2/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
 
-Read [how to contribute.](https://github.com/swapnilsparsh/Rescue/blob/master/Contributing.md)
+9. View the website by pasting the address in your browser.
+```bash
+https://localhost:8000
```
Comment (swapnilsparsh) [inline README.md:80]: Mention that you can open the website by clicking on the link i.e there on the terminal also by adding or option.

Comment (tanishq-arya) [reply README.md:120]: The terminal one is to copy and paste. I'll make it a link.

**Comment by tanishq-arya** [comment]
@swapnilsparsh there were conflicts when I made a fresh PULL. Please check the final readme and let me know if the Linting should be part of installation or it is fine like this only.

**Comment by tanishq-arya** [comment]
@swapnilsparsh Is the linting part of the installation or is it okay like this.

**Review by swapnilsparsh — CHANGES_REQUESTED** [review]
Align the lint and format code that is there.

**Comment by tanishq-arya** [comment]
> Align the lint and format code that is there.

I'll do that. 
But I am asking if it is like this only ( a different section ) or should I add it in steps in continuation with the installation?

**Comment by tanishq-arya** [comment]
image
I am not familiar with linting. So we've to run both the commands. 
I should add `and` between the commands flake and black right?

**Pull request merged**